README.md

```diff
@@ -21,15 +21,27 @@
 
 Sapphire is a Discord bot framework built on top of [discord.js] for advanced and amazing bots.
 
-**Much of the code in this repository is inspired and based on the ideas from [klasa].**
```

Comment:
If you're removing this it would be prevalent to remove the markdown link to Klasa later in the file.